stable/mail2most/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "name" . }}
+  labels:
+    app:  {{ template "name" . }}
```

Comment:
please update to use the new labels, see: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata